runtime/lua/vim/lsp/_tagfunc.lua

```diff
@@ -18,26 +18,71 @@ local function mk_tag_item(name, range, uri, offset_encoding)
   }
 end
 
+--- Returns the Levenshtein distance between the two given string arrays
+--- @param a string[]
+--- @param b string[]
+--- @return number
+local function levenshtein_distance(a, b)
```

Comment:
Just thinking here but I feel like this function might be helpful in other places (perhaps for sorting completions?). We can keep it local to this module for now but it might be a good idea to abstract it later.